org.eclipse.mylyn.discovery-directory/plugin.xml

```diff
@@ -98,82 +98,18 @@
             license="%connectorDescriptor.license.trac"
             name="%connectorDescriptor.name.trac"
             provider="%connectorDescriptor.provider.trac"
-            siteUrl="http://download.eclipse.org/mylyn/snapshots/3.25"
-            statsUrl="http://stats.mylyn.org/discovery/3.25/org.eclipse.mylyn.trac">
+            siteUrl="https://download.eclipse.org/mylyn/releases/latest/">
```

Comment:
Will this bite old releases of Mylyn or are we going to try to be upwards compatible.